src/cpp/polars/TimeSeries.h

```diff
@@ -0,0 +1,143 @@
+//
+// Created by Linda Uruchurtu on 03/10/2018.
+//
+
+#ifndef POLARS_TIMESERIES_H
+#define POLARS_TIMESERIES_H
+
+#include <cassert>
+#include <vector>
+#include <cmath>
+#include <string>
+#include "armadillo"
+#include <chrono>
+#include "date/date.h"
+
+#include "Series.h"
+
+
+namespace polars {
+
+    using namespace std::chrono;
+    typedef std::chrono::duration<double> unix_epoch_seconds;
+
+    template<class TimePointType>
+    class TimeSeries : public Series {
+
+    public:
+        using Series::loc;
+
```

Comment:
We may need to add a load of these to expose all the methods - I think this is the case with templates

Reply:
added todo